src/UI/Settings/Indexers/Options/IndexerOptionsViewTemplate.hbs

```diff
@@ -37,4 +37,15 @@
             <input type="number" name="rssSyncInterval" class="form-control" min="0" max="720"/>
         </div>
     </div>
+    <legend>Availability Options</legend>
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Availability Delay</label>
+	<div class="col-sm-1 col-sm-push-2 help-inline">
+	    <i class="icon-sonarr-form-info" title="A movie will be considered to be available this many days after the Min Availability has been satisfied. (can be negative)"/>
+	    <i class="icon-sonarr-form-info" title="Will not take effect until Movie info is refreshed"/>
```

Comment:
ignore this comment about not taking effect until refreshed.. i dont think that should be the case, if i can implemnet it properly...